src/libs/icon/icon.ts

```diff
@@ -0,0 +1,18 @@
+import { changeIcon as RNChangeIcon } from 'react-native-change-icon'
+
+const DEFAULT_ICON = 'cozy'
+const ALLOWED_ICONS = ['mespapiers']
```

Comment:
I'm not sure this kind of data/config should be in functional files. Though I don't have a suggestion where you could put them. maybe src/libs/icon/config.json for now